### ISSUE TITLE
mesos master & slave enabled at the same time by default

### ISSUE THREAD

**Issue by deric** [deric/puppet-mesos#62]
Latest Mesoshpere packages (at least on Debian 8) automatically enables both `mesos-master` and `mesos-slave` services:
```
# systemctl list-units  | grep mesos
mesos-master.service                               loaded active running   Mesos Master
mesos-slave.service                                  loaded active running   Mesos Slave
```
Although someone might want to run master & slave at the same machine, it's probably not very common setup.

**Comment by deric** [comment]
Experimental implementation in 2b07b2b6240bbf6792d7a7585c90d371980b4902.

**Comment by deric** [comment]
Released in [v0.8.0](https://github.com/deric/puppet-mesos/releases/tag/v0.8.0).
